tests/test_convert.py

```diff
@@ -42,12 +42,12 @@ def test_convert_to_graph(edgelist2, edgelist5):
 
 def test_to_hyperedge_list(edgelist1):
```

Comment:
We should change the definitions of `edgelistX` in `conftest.py` to be sets. Then we won't need all those list comprehensions later on.

Reply:
done.